Cargo.toml

```diff
@@ -18,6 +18,7 @@ integration-tests = []
 name = "tikv_client"
 
 [dependencies]
+regex = "1"
```

Comment:
Could you please use a more generic version? Like `^1` (or probably something better)?

Reply:
I think `1` is fine - it means any version back compat with 1 (i.e., the same as `0.1` for the other deps)